extension/src/popup/views/SendPayment/index.tsx

```diff
@@ -10,26 +10,25 @@ import { SendSettings } from "popup/components/sendPayment/SendSettings";
 import { SendSettingsFee } from "popup/components/sendPayment/SendSettings/TransactionFee";
 import { SendConfirm } from "popup/components/sendPayment/SendConfirm";
 
-// TODO - enforce can't move to next route data not given
```

Comment:
I changed my mind on this, happy to discuss if needed

Reply:
Yeah, this is a bit of an edge case since the user won't have access to a traditional Back/Forward button. It'd be *nice* to have, but might be a bit hairy to enforce. You can see get an idea of how gnarly it can get by looking at Router.tsx

I'm fine to table this until it becomes a problem